math-acceleration-college-algebra-clep/epub.css

```diff
@@ -0,0 +1,51 @@
+/* epub.css */
+
+/* register your embedded fonts as before… */
+@font-face {
+    font-family: 'STIXTwoMath';
```

Comment:
Not sure that this even matters but it took so much finagling I don't want to redo and retest everything every which way to fully understand it right now.